docs-src/dev-guide/compare-sdks.md

```diff
@@ -0,0 +1,403 @@
+---
+id: compare-sdks
+title: What are the differences in each SDK?
+description: A comparison of the features available in each SDK.
+sidebar_label: SDK Comparison
+---
+
+The SDKs for Temporal are constructed as follows:
+
+- Go and PHP derive from the Go SDK.
+- Java derives from the Java SDK.
+- .Net, TypeScript, and Python derive from the Rust Core SDK.
+
+The following table compares the features available in each SDK.
```

Comment:
Provide context onto why we are sharing this table.

Reply:
General table to provide more information on the general use of the SDKs.
It might not be complete, etc, use as is.